spec/lib/mailchimp_spec.rb

```diff
@@ -72,15 +75,57 @@
 																method: 'POST',
 																path: 'lists/list_id/members',
 																body: an_instance_of(String)
-														},
-														{
-																method: 'DELETE',
-																path: 'lists/list_id/members/on_mailchimp'
```

Comment:
Why was this deleted?

Reply:
Must be Github because that change does not appear locally.

Reply:
Perhaps but it needs to be corrected :smile: